### PR TITLE
Trash Cannons

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Adds Trash Cannons, find the book to craft them in maint!

Also, does some improvements to cannon code. Mostly, they use balloon alerts now.

## Why It's Good For The Game

Assistants thermiting the wall is sooooo outdated. Now we're using funny cannons to do it

## Changelog
:cl: tralezab code, paxilmaniac sprites
expansion: Added Trash Cannons. Find the recipe in maint!
/:cl:

### CODE REVIEW

File: code/game/objects/structures/cannons/trash_cannon.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/obj/structure/cannon/trash
+	name = "trash cannon"
+	desc = "Okay, sure, you could call it a toolbox welded to an opened oxygen tank cabled to a skateboard, but it's a TRASH CANNON to us."
+	icon_state = "falconet_patina"
```
Comment (Paxilmaniac) [inline code/game/objects/structures/cannons/trash_cannon.dm:4]: :thinking:

**Comment by SmArtKar** [comment]
Seems really nice but uh, maybe you should make trash cannon have like a chance to explode? Because right now it's just a normal cannon with no downsides

File: code/game/objects/structures/cannons/trash_cannon.dm (hunk 1 above)
Comment (tralezab) [reply code/game/objects/structures/cannons/trash_cannon.dm:4]: oops

**Comment by tralezab** [comment]
Alright you asked for it though

**Comment by MrDoomBringer** [comment]
thread theme https://www.youtube.com/watch?v=s6C-6WCamGw

**Comment by Paxilmaniac** [comment]
no mention that i made the sprites smh smh

**Comment by MrDoomBringer** [comment]
wtf spriters rise up

**Comment by tralezab** [comment]
SORRY I FORGOT

**Comment by Paxilmaniac** [comment]
> SORRY I FORGOT

i forgor :(

**Comment by Mothblocks** [comment]
sounds funny

**Pull request merged**